resources/views/livewire/toast.blade.php

```diff
@@ -1,4 +1,4 @@
-<div class="grid fixed right-0 bottom-0 z-20 grid-rows-1 gap-2 mr-1 mb-5" style="min-width: 250px; max-width: 350px">
+<div class="grid fixed right-0 bottom-0 z-50 grid-rows-1 gap-2 mr-1 mb-5" style="min-width: 250px; max-width: 350px">
```

Comment:
what was this for?

Reply:
Possibly so it's above modals? I noticed that they appear behind the modal overlay but didn't really acknowledge as an issue since I was looking for something else 🤔

Reply:
@ItsANameToo yeah is to ensure the toast is over the modal (and everything)